docs/content/v0.20/architecture.md

```diff
@@ -309,40 +310,20 @@ follows:
     -  Iterates on all binding-policies, and for each of them:
         - Evaluates whether the object matches the downsync selection 
           criteria in the `BindingPolicy`.
-        - If there is a match:
-          - Notes the object as a match for the in-memory representation of the relevant `Binding`.
-            - Enqueues the representation of the relevant `Binding` for syncing if changed. 
-          - Lists ManagedClusters and finds the matching clusters using the label selector expression for clusters.
-            - If there are matching clusters, adds the names of the cluster
-              to a hashmap setting the name of the cluster as a key. Cluster
-              groups from different bindind policies are merged together.
-          - If any of the matched `BindingPolicy` has ` WantSingletonReportedState`
-            set to true, clusters are sorted in alphanumerical order and
-            only the first cluster is selected for delivery. Note that setting 
-            `WantSingletonReportedState` in one of the binding-policies that 
-            matches the object affects the behavior for all matching binding-policies, 
-            those with matching clusters and those w/o matching clusters.
-        - If there are no matching clusters, the worker returns without 
-       actions and is ready to process other events from the queue. 
-        - If there are matching clusters:
-          - Wraps the object into a ManifestWork
-       - Adds a label for each matched `BindingPolicy` to the ManifestWork that 
-         is used to track the `BindingPolicy` that caused the object to be
-         delivered to one or more clusters. The label contains both the 
-         `BindingPolicy` name (note that `BindingPolicy` is cluster-scoped) and the WDS name.
-         This way, when a binding-policy is deleted or updated it is possible to 
-         locate the associated ManifestWorks for deletion or label removal 
-         (if more than one label is present, as other binding-policies may “own” the object).
-       - For each cluster:
-         -  Sets the manifest namespace == name of the cluster
-         -  Uses the client connected to the ITS to do a server-side
-            apply patch of the manifest into the namespace.
-         - At this time there is only one field manager name for the server-side
-           apply (`kubestellar`) but the name should include also the WDS
-           name to allow detecting conflicts when two different field managers
-           try to patch a manifest with the same name.   
+        - Whether the object is a match or not, the worker notes the object as a match for the in-memory representation 
```

Comment:
"the object is ... not" clashes with "notes the object as a match". I think you are trying to say something like the following?

> The worker notes whether or not the object is a match, in an internal representation of the matching set.